dex/README_DOCKER.md

```diff
@@ -0,0 +1,90 @@
+## 7. Matcher Server & Matcher Node in Docker
```

Comment:
How a user can change JVM settings (e.g. the max heap size?)

Reply:
Done